### DISCUSSION TITLE
Missing wingpanel dep

### DISCUSSION THREAD

**Discussion by ChildishGiant** [elementary/panel-sound#208] [Q&A] [answered]
I have `libwingpanel-2.0-dev` installed (along with `libwingpanel-2.0-0` and `wingpanel`) but meson can't find any version of wingpanel. `meson.build:30:0: ERROR:  Dependency "wingpanel" not found`. Is there a certain package I need that I can't seem to find?

**Comment by davidmhewitt** [comment, marked as answer]
For elementary 6.0 the wingpanel package with the versioned name was changed to an unversioned name. You could try manually reverting [this commit](https://github.com/elementary/wingpanel-indicator-sound/commit/845e631c420cca84a4807ec08f201be439a9ae88) and seeing if it still compiles against the older wingpanel library. But the latest commit of most repositories now targets elementary OS 6.0 and some things won't compile at all with older library versions.

**Reply by ChildishGiant** [reply]
Ah, I'm still running Hera on my main machine (as recommended :wink:) so that'll be why. I had to also revert [this line](https://github.com/elementary/wingpanel-indicator-sound/blame/301757cedcc4875804e9944380f34d7fa5c20204/meson.build#L54). I'm now running in to build errors. Running [v2.1.8](https://github.com/elementary/wingpanel-indicator-sound/releases/tag/2.1.8) solves all these issues  but I imagine if I'm attempting to work on this project at all I should really be doing it on Odin so I don't need to then update it all?

**Reply by davidmhewitt** [reply]
Indeed. It will just compile out of the box on Odin.

You can either develop on top of the latest commit that compiles on Hera, and then rebase your work on top of master once you're comfortable running Odin. That may be a bit of a pain, since there's been a reasonably significant amount of work on audio device selection in the indicator since the stable release.

Or, you can just develop on Odin beta. But the usual caveats about it not being production ready and not upgradable to the stable release once it's out apply.
